crates/notion-core/src/version/mod.rs

```diff
@@ -163,3 +167,17 @@ pub mod option_version_serde {
     }
 
 }
+
+pub mod lts_version_serde {
+    use serde::{self, Deserialize, Deserializer};
+
+    pub fn deserialize<'de, D>(deserializer: D) -> Result<bool, D::Error>
+    where
+        D: Deserializer<'de>,
+    {
+        match String::deserialize(deserializer) {
```

Comment:
This looks perfect! The only comment I have is that I think this fits better with the parsing logic in `inventory/serial.rs`, as opposed to here with the version parsing. I believe it can also be a non-`pub` function in that module that is referenced directly in `deserialize_with`, instead of needing its own module.